earth_enterprise/src/fusion/autoingest/tools/geaddtoimageryproject.cpp

```diff
@@ -35,20 +35,40 @@ usage(const std::string &progn, const char *msg = 0, ...)
     fprintf(stderr, "\n");
   }
 
-  fprintf(stderr,
-          "\nusage: %s [options] -o <projectname> {[--maxlevel <level>] "
-          "<insetresource>}...\n"
-          "   Supported options are:\n"
-          "      --help | -?:      Display this usage message\n"
-          "      --mercator : use mercator projection\n"
-          "      --flat : use flat(Plate Carre) projection (default)\n"
-          "      --historical_imagery : change this project to be a historical\n"
-          "                             imagery project\n"
-          "      --no_historical_imagery : make this a normal project\n"
-          "                         (i.e., not a historical imagery project)\n"
-          "   By default, new projects are NOT time machine projects"
-          " unless --historical_imagery is specified.\n",
-          progn.c_str());
+  if (AssetDefs::Terrain == AssetType){
```

Comment:
I created a new terrain project using `genewterrainproject -o terrainprojecttest2 --terrain_overlay --start_level 4 --resource_min_level 4`: no error but when i opened the terrain project in fusion "Build Overlay" checkbox was not checked and the start and min levels values were the default ones: 4 and 12. Is this is expected ?

Reply:
@tst-nfarah I did a complete rebuild with the latest commit, and both of these commands work for me as expected. Let me know if you're seeing something different.

Reply:
@tst-sbayard I rebuilt then i run `genewterrainproject -o terrainproject9 --terrain` and looked at the project in Fusion UI. It did have `Build "Overlay"` checkbox enabled ! Are we not looking for the full `terrain_overlay` and only partially to decide if overlay to be on/off ?

Reply:
I'm seeing the same behavior. I think this may be a bug in the option parsing code. On a hunch, I executed this command `genewimageryproject -o imageryproject --hist` and sure enough, it created an imagery project with "Support Historical Imagery" enabled. I'm going to try the same thing on a 5.2.4 release build to verify that this was present in previous versions. If it already existed, I can take a quick look, but that may need to be filed as a separate bug.

Reply:
After further research, this seems to be by design. The option parsing code calls `getopt_long_only` which is a standard library function. The documentation states that 

> Long-option names can be abbreviated if the abbreviation is unique or an exact match for some defined option

To test, I executed `grep -r getopt_long_only .` in the opengee repository. Then I ran `grep --recurs getopt_long_only .`, using `--recurse` instead of `--recursive`. That produced the exact same results as the first call. 

I had no idea, but apparently that's just a standard feature.

Reply:
@tst-sbayard it looks like this was intended to work that way. I guess the idea is to accommodate with close enough strings.